app/src/main/java/app/revanced/integrations/shared/Utils.java

```diff
@@ -233,6 +233,11 @@ public static Context getContext() {
         return context;
     }
 
+    public static void setContext(Context appContext) {
+        context = appContext;
+        Logger.printDebug(() -> "Set context: " + appContext); // Cannot log before context is set.
```

Comment:
Is that the case? The method does not rely on context: https://github.com/revanced/revanced-integrations/blob/bedcef89a5df3846e6613fa216a8cb79143bbedf/app/src/main/java/app/revanced/integrations/shared/Logger.java#L60-L75